package.json

```diff
@@ -47,6 +47,7 @@
     "peerDependencyRules": {
       "ignoreMissing": [
         "@algolia/client-search",
+        "playwright",
```

Comment:
This is needed as [`axe-playwright`](https://www.npmjs.com/package/axe-playwright) has a peer dependency of `playwright` which we explicitly don't use in favor of `@playwright/test`. Both official packages are [different](https://playwright.dev/docs/library) in the sense that `playwright` automatically installs browser binaries while `@playwright/test` does not. Running `pnpm test:a11y` in the `packages/starlight/` directory will take care of installing the necessary dependencies in our case only when needed.